docs/src/main/sphinx/client/jdbc.md

```diff
@@ -23,14 +23,12 @@ Download {maven_download}`jdbc` and add it to the classpath of your Java applica
 
 The driver is also available from Maven Central:
 
-```{eval-rst}
-.. parsed-literal::
-
-    <dependency>
-        <groupId>io.trino</groupId>
-        <artifactId>trino-jdbc</artifactId>
-        <version>\ |version|\ </version>
-    </dependency>
+```xml
+<dependency>
+    <groupId>io.trino</groupId>
+    <artifactId>trino-jdbc</artifactId>
+    <version>|trino_version|</version>
```

Comment:
Make sure this still gets replaced correctly in the rendered docs

Reply:
It does ...